### PR TITLE
Added view-list

### PR DESCRIPTION
**Pull request by TLMatthies**
Can now view a list's content. Currently only returns name of food item and quantity of.
(Also my delete list endpoint is in there too, on accident)

### CODE REVIEW

File: src/api/users.py [hunk 1]
```diff
@@ -293,4 +293,86 @@ def get_list_history(user_id: int):
...
+        
+        return_list = []
+        for row in data:
+            return_list.append(
+                {
+                    "name": row.name,
+                    "quantity": row.quantity
+                }
+            )
+        return return_list
```
Comment (kevinrutledge) [inline src/api/users.py]: A more pythonic way to implement:

```python
return_list = [
    {"name": row.name, "quantity": row.quantity}
    for row in data
]
return return_list
```

or

```python
return [
    {"name": row.name, "quantity": row.quantity}
    for row in data
]
```

**Comment by kevinrutledge** [comment]
Nothing major to add. It's good.

File: src/api/users.py (hunk 1 above)
Comment (TLMatthies) [reply src/api/users.py]: I like your list comprehension method, I always forget about them. Updated it to act like your second example.

**Review by Mcray3000 — APPROVED** [review]
Looks good

**Review by Mcray3000 — APPROVED** [review]

**Pull request merged**